### DISCUSSION TITLE
How many plots does this do?

### DISCUSSION THREAD

**Discussion by LogenP** [ericaltendorf/plotman#261] [Q&A] [answered]
I can't seem to figure out how many plots does the system do once running? it will just go until the drive is completely full? what if I add drives/changer ssd cards mid plotting? how would that work?

**Comment by ericaltendorf** [comment, marked as answer]
Plotman runs continuously, continuously starting new plot jobs to the 'dst' drives and continuously moving those plots to whatever disks it can find with free space on the farmer (archiving).  You can add drives on the farmer and plotman will just keep filling whatever is there.  You can just keep buying drives forever. :)  Adding tmp or dst drives is also theoretically possible.  Removing drives is more delicate since you'd need to make sure they're not in use by any in-flight jobs.

Side note: if you just want a simple fixed number of plots to a single drive, you should probably just run the plotter yourself; Plotman wouldn't be doing much for you.

**Reply by LogenP** [reply]
Understood. So I just chnage the config file to more destinations if I add additional storage?
Do I need to stop the plotting before doing that? Also, I'm going to be upgrading to a different nvme ssd card, how would I do that? Stop plotter and then restart?
What about changing we are both such as Max plots and so forth, need to stop the process and restart or just chnage the file and it will update?

Thanks for the insight!

**Reply by BasilHorowt** [reply]
@LogenP you must restart plotman whenever you change config file.  Check Readme and [FAQ](https://github.com/ericaltendorf/plotman/wiki/FAQ) for more basic info
